README.md

```diff
@@ -3,14 +3,19 @@ Automation Tools
 
 The Automation Tools project is a set of python scripts, that are designed to automate the processing of transfers in an Archivematica pipeline.
 
-Currently, the only automation tool is automate transfers.  It is used to prepare transfers, move them into the pipelines processing location, and take actions when user input is required.  Only one transfer is sent to the pipeline at a time, the scripts wait until the current transfer is resolved (failed, rejected or stored as an AIP) before automatically starting the next available transfer. 
+
+Automated Transfers
+------------------
```

Comment:
We used dashes elsewhere in the document for h1 and h2, so I continued to use == and -- for the first two levels, and ### for the rest.  Markdown accepts either (https://daringfireball.net/projects/markdown/basics).  It's also a more obvious visual header than just ### prepended.